### ISSUE TITLE
Unable to update series in map (v2.4)

### ISSUE THREAD

**Issue by flandrade** [highcharts/highcharts-angular#141]
> For general tech support, please see www.highcharts.com/support.
> Please report only issues about `highcharts-angular` wrapper or content of this repository.
> For general issues with Highcharts and TypeScript the correct repository to report such issues is [main Highcharts repository](https://github.com/highcharts/highcharts/issues).

### Describe the bug
The map gets empty after updating the data series (Highmaps). Updating the map works in `v2.3`, but the issue appears with `v2.4.0`. This is a code snippet of the operation:  

```ts
this.chartMap.series = [{
  data: [
     ["fo", 0],
     ["um", 1],
     ["us", 2],
     ["jp", 3],
   ]
 } ];

this.updateFlag = true;
```

### Expected behavior
The map shouldn't be empty. It should show the new data series. 

### Demo
Here you can find a demo of [this issue](https://codesandbox.io/s/highmaps-v-24-0uwqr) with version `v2.4`. In addition, you can find a [working example](https://codesandbox.io/s/highmaps-v-201-mgvol) in version `v.2.3`. 

In both, the code is the same (based on the [official Demo #1 and Demo #3](https://github.com/highcharts/highcharts-angular/blob/master/src/app/app.module.ts)). The only difference is the `highcharts-angular` version. 

#### Steps required to recreate the problem in the demo:
1. [Click in](https://codesandbox.io/s/highmaps-v-24-0uwqr) the button: "Update Chart" 
2. The map gets empty

### Setup used
 - **NodeJS version:** v10.13.0
 - **Angular version:** v7.2 (my machine), v5.2 (in Demo)
 - **TypeScript version:** v3.4.3 (my machine), v2.5.4 (in Demo)
 - **highcharts-angular version:** v2.4 (my machine), v2.4 (in Demo)
 - **highcharts:** v7.1.1 (my machine), v7.1.1 (in Demo)

**Comment by KacperMadej** [comment]
Hi @flandrade 

Thank you for reporting about the problem.

The problem is in Highcharts core. If you could change the Highcharts package version in the working demo to use the latest 7.1.1 version you will get the same results no matter what version (2.0.1 or 2.4.0) of the wrapper is being used.

I'll debug this, create an Angular-free demo, report this on the main Highcharts repository, try to track the offending commit, try to provide workaround better than "use older version of code", and get back here to close this issue in favor of the one on the main HC repo.

**Comment by KacperMadej** [comment]
Looks like the issue is fixed on master branch of Highcharts (next release candidate) - http://jsfiddle.net/BlackLabel/Lkj2w8b3/

(Here's a simple broken demo: http://jsfiddle.net/BlackLabel/Lkj2w8b3/1/)

ETA for the next version of Highcharts (7.1.2) is next week, so let's wait for it.

**Comment by KacperMadej** [comment]
Fixed in 7.1.2 - if not, please comment and the issue could be reopened.

**Comment by RonjaKnudtsen** [comment]
This works for me now, thank you for updating us on the issue
